### ISSUE TITLE
using pertrained weight, but raise a RuntimeError

### ISSUE THREAD

**Issue by shiguangliuguo** [ZhendongWang6/Uformer#11]
hello! thanks for your devotion.
I train the Uformer using the SIDD on 2 V100 as you suggestion.
i trained nearly 69 epoches and stoped it. i got a weight file.
i valid it and it perform well.
but when i want  to fintune on 2 V100,  i add --resume command line.

in train.py line 169: loss_scaler(loss, optimizer, parameters=model_restoration.parameters())
i raise a RuntimeError:Expeted all tensors to be on the same device, but found at least two devices, cude:0 and cpu 

i dont know how to solve this matter, have you met this problem?
thx!

**Comment by vinthony** [comment]
Hi, this bug has been fixed, please check out our new code in https://github.com/ZhendongWang6/Uformer/blob/97c8f95cac035f5cc84536486c27c78dde12023c/train.py#L79-L97

**Comment by shiguangliuguo** [comment]
> Hi, this bug has been fixed, please check out our new code in
> 
> https://github.com/ZhendongWang6/Uformer/blob/97c8f95cac035f5cc84536486c27c78dde12023c/train.py#L79-L97

ohhh, it really works.
i got to say its wired when facing such bug.
thanks for your improvment!

**Comment by aiaini66** [comment]
@shiguangliuguo  

> valid it and it perform well.
hello, as you said you get a well perform, Did you train naked from scratch, or did you load the author's open source pre-trained model for fine-tuning?
